src/tests/rnp_tests.c

```diff
@@ -156,13 +145,132 @@ main(int argc, char *argv[])
         tests[i].teardown_func = teardown_test;
     }
 
-    int ret = 0;
-    for (int i = 0; i < iteration; i++) {
-        printf("Iteration %d\n", i);
+    if (iterations == 0) {
         ret = cmocka_run_group_tests(tests, setup_test_group, teardown_test_group);
-        if (ret != 0) {
-            break;
+    } else {
```

Comment:
Possibly it was error during the merge, but I didn't want to change anything around this. What's wrong with this logic?